lib/strings/text.php

```diff
@@ -0,0 +1,53 @@
+<?php
+declare(strict_types=1);
+
+
+namespace jsys\types\strings;
+
+class Text implements TextInterface
+{
+    private $str;
+    public function __construct(string $str)
+    {
+        {
+            $this->str = $str;
+        }
+    }
+
+    public function isEmpty(string $str):bool
+    {
+        if (!empty($str))
+        {
+            throw new \InvalidArgumentException("Non empty '$str' is not Allowed", 1021);
+
+        }
+        return true;
+    }
+
+    public function length(string $str):int
+    {
+        return mb_strlen($str);
+    }
+    public function hasSpaces(string $str)
+    {
```

Comment:
nope
is Space and has space are two different  functions

Reply:
can you fix it?